functions/src/types/Room.ts

```diff
@@ -0,0 +1,10 @@
+export type RoomId = string
+export type RoomSid = string
+
+export interface RoomParameters {}
+
+export interface Room {
+    roomId: RoomId
+    roomSid: RoomSid
```

Comment:
may I ask why to type differently every string, to be more specific on what it means, considering the variable name will already hold that information ? I have the impression that, as a developer who reads it, I will understand the kind of information from the variable name already, but I'll have to go look into the `RoomId` type, to understand it is a simple `string`.

Also I find that the names are repeated. Why not to simplify with:

```typescript
export interface Room {
  id: string
  sid: string
}
```

so that when we'll access that info, it will be as `room.id` instead of `room.roomId`. In case you want to get the same name from destructuring, it will still be possible:

```typescript
const {id: roomId} = room
```

Reply:
ensuring that a given parameter is of a specific type ensure that a given id is NOT wrongfully passed to a given parameter. 
Eg: RoomId will not be used as a RoomSid. A pontential counter argument will be that Typescript does not check for that specific use case, so I believe in this case we could use type Branding : https://stackoverflow.com/a/49432424/1377145

If we go for type branding (we probably should), indeed the class prefix is not needed.

Reply:
Interesting, I'll read a little more on the subject